vmm/src/vm_config.rs

```diff
@@ -154,6 +154,12 @@ pub struct MemoryZoneConfig {
     pub hotplugged_size: Option<u64>,
     #[serde(default)]
     pub prefault: bool,
+    #[serde(
+        default,
+        serialize_with = "serialize_memory_zone_fd",
+        deserialize_with = "deserialize_memory_zone_fd"
```

Comment:
I think in #7281 we decided against a special serialize path. See https://github.com/cloud-hypervisor/cloud-hypervisor/pull/7281/commits/666444d333045e38ead1c82cf35ddbe181f85614.

I'd love to have it consistent here as well.